client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
```

Comment:
Sure — is that common for this kind of thing in Calypso? I can see the value (keep it out of public source, easier to change without deploying Calypso code) but it also adds an extra HTTP request, and I don't imagine it will change frequently.

Also wondering if you're thinking the whole config object would be in the API request, or just the `id`?